ShopLens/VoiceRecognition/VoiceRecognitionTest.cs

```diff
@@ -65,37 +71,45 @@ public void SpeechRecognizer_SpeechRecognized(object sender, SpeechRecognizedEve
 
             if (speechResult == helloCmd)
```

Comment:
Possible way to make this code shorter - use 'switch' statement and assign the response to a variable, and just use that assigned variable at the end (only one place where you call `.Text +=` and `.SayMessage` this way. Something like this:

```csharp
string response = "";
switch (speechResult):
{
    case(helloCmd):
        response = helloRsp;
        break;
     ....
}

CommandOutputBox.Text += response;
textVoicer.SayMesage(response);

Reply:
That's a good point, I'll change it.